### PR TITLE
sql: update transaction document by morgan's suggestions

### PR DESCRIPTION
**Pull request by jackysp**
Signed-off-by: Shuaipeng Yu <jackysp@gmail.com>
https://github.com/pingcap/docs/pull/1161

### CODE REVIEW

File: sql/tidb-specific.md [hunk 1]
```diff
@@ -323,6 +323,8 @@ set @@global.tidb_distsql_scan_concurrency = 10
 
 这个变量用来设置是否禁用显式事务自动重试，设置为 1 时，不会自动重试，如果遇到事务冲突需要在应用层重试。
 
+设置这个变量为 0，意味着 TiDB 将会自动重试事务，这样可以在事务提交时遇到更少的错误。需要注意的是，这样将会导致数据的丢失更新异常。
```
Comment (lilin90) [inline sql/tidb-specific.md]:
```suggestion
如果将该变量的值设为 0，TiDB 将会自动重试事务，这样在事务提交时遇到的错误更少。需要注意的是，这样可能会导致数据更新丢失。
```

File: sql/transaction-isolation.md [hunk 2]
```diff
@@ -53,11 +53,11 @@ MySQL 可重复读隔离级别在更新时并不检验当前版本是否可见
 
 ## 事务重试
 
-执行失败的事务可能会由 TiDB 自动重试，但这可能会导致更新丢失。通过设置 `tidb_disable_txn_auto_retry = 0` 可开启该项功能，同时要注意 `tidb_retry_limit` 的值不能为 0，否则，也会禁用自动重试。
+TiDB 默认不会进行事务重试，因为重试事务将会导致丢失更新异常。如果应用程序可以容忍事务重试带来的异常，或者说并不关注事务是否以 SI 隔离级别来执行，可以开启自动重试。通过设置 `tidb_disable_txn_auto_retry = 0` 可开启该项功能，同时要注意 `tidb_retry_limit` 的值不能为 0，否则，也会禁用自动重试。开启自动重试以后，事务遇到提交出错的可能性会降低。
```
Comment (lilin90) [inline sql/transaction-isolation.md]:
```suggestion
TiDB 默认不会进行事务重试，因为重试事务可能会导致更新丢失。如果业务可以容忍事务重试导致的异常，或者说并不关注事务是否以 SI 隔离级别来执行，可以开启自动重试。通过设置 `tidb_disable_txn_auto_retry = 0` 可开启该项功能，同时要注意 `tidb_retry_limit` 的值不能为 0，否则，也会禁用自动重试。开启自动重试以后，事务遇到提交出错的可能性会降低。
```

File: sql/transaction-model.md [hunk 3]
```diff
@@ -64,4 +64,4 @@ COMMIT;
 
     > **注意：**
     >
-    > `LOAD DATA` 在 TiDB 中没有事务原子性的保证，不建议在生产环境中使用。
+    > `LOAD DATA` 在 TiDB 中没有事务原子性的保证，可以在数据初次导入时使用，不建议在生产环境的应用中使用。
```
Comment (lilin90) [inline sql/transaction-model.md]:
```suggestion
    > `LOAD DATA` 在 TiDB 中没有事务原子性的保证，可以在数据初次导入时使用，不建议在生产环境中使用。
```

**Comment by huangxiuyan** [comment]
@jackysp PTAL and update, thanks~

**Comment by lilin90** [comment]
@jackysp Please address comments. Thanks!

**Comment by jackysp** [comment]
Addressed, PTAL @huangxiuyan @lilin90 .

File: dev/reference/transactions/transaction-model.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ aliases: ['/docs-cn/sql/transaction-model/']
 
 # 事务模型
 
-TiDB 使用乐观事务模型，在执行 `Update`、`Insert`、`Delete` 等语句时，只有在提交过程中，执行 `Update`，`Insert`，`Delete` 等语句时才会检查写写冲突，而不是像 MySQL 一样使用行锁来避免写写冲突。类似的，诸如 `GET_LOCK()` 和 `RELEASE_LOCK()` 等函数以及 `SELECT .. FOR UPDATE` 之类的语句在 TiDB 和 MySQL 中的执行方式并不相同。所以业务端在执行 SQL 语句后，需要注意检查 COMMIT 的返回值，即使执行时没有出错，COMMIT 的时候也可能会出错。
+TiDB 使用乐观事务模型，在执行 `UPDATE`、`INSERT`、`DELETE` 等语句时，只有在提交过程中，执行 `UPDATE`，`INSERT`，`DELETE` 等语句时才会检查写写冲突，而不是像 MYSQL 一样使用行锁来避免写写冲突。类似的，诸如 `GET_LOCK()` 和 `RELEASE_LOCK()` 等函数以及 `SELECT .. FOR UPDATE` 之类的语句在 TIDB 和 MYSQL 中的执行方式并不相同。所以业务端在执行 SQL 语句后，需要注意检查 `COMMIT` 的返回值，即使执行时没有出错，`COMMIT` 的时候也可能会出错。
```
Comment (lilin90) [inline dev/reference/transactions/transaction-model.md]:
```suggestion
TiDB 使用乐观事务模型，在执行 `UPDATE`、`INSERT`、`DELETE` 等语句时，只有在提交过程中，执行 `UPDATE`，`INSERT`，`DELETE` 等语句时才会检查写写冲突，而不是像 MySQL 一样使用行锁来避免写写冲突。类似的，诸如 `GET_LOCK()` 和 `RELEASE_LOCK()` 等函数以及 `SELECT .. FOR UPDATE` 之类的语句在 TiDB 和 MySQL 中的执行方式并不相同。所以业务端在执行 SQL 语句后，需要注意检查 `COMMIT` 的返回值，即使执行时没有出错，`COMMIT` 的时候也可能会出错。
```

**Comment by lilin90** [comment]
@jackysp Please update it according to my new comment.

**Comment by jackysp** [comment]
PTAL @lilin90

**Pull request merged**